client/my-sites/earn/customers/index.tsx

```diff
@@ -0,0 +1,344 @@
+import { Card, Button, Dialog, Gridicon } from '@automattic/components';
```

Comment:
For reviewers: None of the code in this file is new. Every line is directly copied/moved from memberships/index.